### PR TITLE
Reuseable workflow build unit test

### PR DESCRIPTION
**Pull request by FleetAdmiralButter**
1) Frontend build and test workflow

### CODE REVIEW

File: .github/workflows/build_unit-test.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
+name: Reusable CI/CD Workflow
```
Comment (nicksantamaria) [inline .github/workflows/build_unit-test.yml:1]:
```suggestion
name: Reusable workflow for ripple builds
```

File: .github/workflows/run_db_trigger.yml [hunk 2]
```diff
@@ -0,0 +1,135 @@
+name: Reusable View MySQL Tables Workflow
```
Comment (nicksantamaria) [inline .github/workflows/run_db_trigger.yml:1]:
```suggestion
name: Reusable workflow to build MySQL database images 
```

File: .github/workflows/run_db_trigger.yml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        required: true
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      SLACK_WEBHOOK_URL:
+        required: true
+      
+jobs:
+  view_tables:
```
Comment (nicksantamaria) [inline .github/workflows/run_db_trigger.yml:40]: Can we rename the job to something more meaningful?

File: .github/workflows/run_db_trigger.yml [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+          status: ${{ needs.view_tables.result }}
+          text: 'Attention: The db-trigger workflow has FAILED in project "${{ inputs.project }}" for repo "${{ github.repository }}"!'
+          fields: 'project: ${{ inputs.project }}, repo: ${ { github.repository }}'
+          mention: 'here'
+          author_name: 'DPC TEAM'
+          channel: '#dpc-devops-internal'
+          color: 'danger'  
+          if_mention: 'always'
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```
Comment (nicksantamaria) [inline .github/workflows/run_db_trigger.yml:117]: This should be a slack webhook for the sdp slack org.

File: .github/workflows/run_db_trigger.yml [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+      - name: Send Notification to Slack on Failure
+        if: ${{ needs.view_tables.result == 'failure' }} 
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ needs.view_tables.result }}
+          text: 'Attention: The db-trigger workflow has FAILED in project "${{ inputs.project }}" for repo "${{ github.repository }}"!'
+          fields: 'project: ${{ inputs.project }}, repo: ${ { github.repository }}'
+          mention: 'here'
+          author_name: 'DPC TEAM'
+          channel: '#dpc-devops-internal'
```
Comment (nicksantamaria) [inline .github/workflows/run_db_trigger.yml:113]: Need this updated to `#dpc-events`

**Review by nicksantamaria — CHANGES_REQUESTED** [review]

**Review by GROwen — CHANGES_REQUESTED** [review]
build_unit-test LGTM @FleetAdmiralButter but could you fix up the line spacing between steps please, just one line is sufficient and it'd be good if they're consistent.

**Review by GROwen — APPROVED** [review]

**Pull request merged**